Core/clim-basic/windowing/events.lisp

```diff
@@ -146,40 +176,6 @@
             (pointer-event-x event)
             (pointer-event-y event))))
 
-(defmacro get-pointer-position ((sheet event) &body body)
-  (alexandria:once-only (sheet event)
-    `(multiple-value-bind (x y)
-         (if ,sheet
-             (untransform-position (sheet-delta-transformation ,sheet (graft ,sheet))
-                                   (device-event-native-graft-x ,event)
-                                   (device-event-native-graft-y ,event))
-             (values (device-event-native-x ,event)
-                     (device-event-native-y ,event)))
-       (declare (ignorable x y))
-       ,@body)))
-
-(defmethod pointer-event-x ((event pointer-event))
-  (get-pointer-position ((event-sheet event) event) x))
-
-(defmethod pointer-event-y ((event pointer-event))
-  (get-pointer-position ((event-sheet event) event) y))
-
-(defgeneric pointer-event-position* (pointer-event))
-
-(defmethod pointer-event-position* ((event pointer-event))
-  (get-pointer-position ((event-sheet event) event)
-    (values x y)))
```

Comment:
This has been removed, right? If the removal was on purpose, please mention it in the commit message (preferably with a justification).

Reply:
fixed

I've added to the commit message the following:

```
We also remove a generic function POINTER-EVENT-POSITION* which was returning
the event coordinates in sheet coordinate system /at the time of calling it/.
This function is not used in McCLIM codebase and is not exported.
```